pkg/rule/rule_manager.go

```diff
@@ -97,6 +97,15 @@ func (r *ManagedRules) Start(ctx context.Context) error {
 				r.logger.Warnf("error sending manager_loop_update_data metric: %v", err)
 			}
 
+			rules, err := r.ruleTarget.Get(r.conn)
```

Comment:
It might make sense to emit the metrics first. If only to make sure we emit them even if something errors out later so we don't end up with gaps in metrics if there are problems with `rulesUpdateFunc()`.